docpages/INDEX.md

```diff
@@ -48,8 +48,8 @@ You can find further releases in other architectures and formats or the source c
 ### Linux
 The library runs ideally on **Linux**.
 
-### Mac OS X and FreeBSD
-The library is well-functional and stable on **Mac OS X** and **FreeBSD** too.
+### Mac OS X, FreeBSD, and OpenBSD
```

Comment:
last element in a list doesnt have "," after it, this is the "oxford comma" which nobody really uses outside of universities.

Reply:
We have it in nearly every other place now, removing it will start to form inconsistencies